mindsdb/integrations/handlers/bard_handler/__init__.py

```diff
@@ -0,0 +1,21 @@
+from mindsdb.integrations.libs.const import HANDLER_TYPE
+from mindsdb.utilities import log
+
+logger = log.getLogger(__name__)
+
+from .__about__ import __description__ as description
+from .__about__ import __version__ as version
+
+try:
+    from .bard_handler import BardHandler as Handler
```

Comment:
As discussed above, please rename the handler to avoid using `Bard` because it is not accurate.

Reply:
yes sure